### PR TITLE
[CS] Cleanup of common aliases

### PR DESCRIPTION
**Pull request by zbeky**
Cleanup of common aliases
More general weather preposition
More specific vacuum commands and responses

## Summary by CodeRabbit

- **New Features**
	- Enhanced responses for vacuum commands, providing personalized feedback (e.g., including vacuum names).
	- Expanded command recognition for vacuum intents, allowing for more varied user inputs.
	- Improved sentence structures for weather-related commands.

- **Bug Fixes**
	- Corrected grammatical structures in sensor-related responses.

- **Chores**
	- Updated context exclusions for certain commands to prevent unintended triggers.

### CODE REVIEW

**Comment by zbeky** [comment]
@coderabbitai resolve

File: sentences/cs/_common.yaml [hunk 1]
```diff
@@ -331,26 +334,25 @@ expansion_rules:
...
-  je_jsou: "(je|jsou)"
-  v: "(v|ve|na)"
-  y_ej: "(ý|ej)"
-  y_e: "(ý|é)"
+  zatahnout: "(za|s)táhn(out|i)"
+  prejit: "přej(ít|di)"
   koncentrace: "(úroveň|množství|koncentrace)"
-  jaky_je: "(jak<y_e> (je|bude)|jaká (je|bude)|jakou|jaká|jak|kolik|kolik (je|bude))"
+
+  y_e: "(ý|ej|á|é|ých|ejch)"
```
Comment (halecivo) [inline sentences/cs/_common.yaml:345]: I hope that this grouping of too many endings (which will produce lot of incorrect forms like dešťovéch srážek) won't increase error rate in detection.

**Review by halecivo — APPROVED** [review]

**Comment by zbeky** [comment]
Bump. Should we try the more general suffixes and revert if the STT engine is then creating false positives?

**Comment by halecivo** [comment]
yep, we can try

**Pull request merged**